packages/frontend/core/src/utils/flatten-object.ts

```diff
@@ -0,0 +1,35 @@
+export function flattenObject(ob: any, target: any = {}, prefix = '') {
```

Comment:
how about array?

Reply:
I think string is more intuitive here

Reply:
I mean what if value of `ob[key]` is an array

Reply:
I deleted this function because we don't actually use it. We actually use
https://github.com/toeverything/AFFiNE/blob/8982886440069c44b1dbf24832eb3b249739b490/packages/frontend/core/src/modules/editor-settting/schema.ts#L41

It will stop flatten when it encounters any non-zod.object element.